foundation-model-stack/fms-hf-tuning
# feat: install fms-acceleration to enable qlora

### Description of the change
- add flag `ENABLE_FMS_ACCELERATION` to install fms-acceleration, this is currently set to false as we are testing but I believe we want to install fms-acceleration by default
- fms-acceleration required installing `g++` and `python3.11-devel` deps


### Related issue number



### How to verify the PR



Built image and able to run qlora on it with flag `--auto-gptq triton_v2` or passing in JSON `"auto-gptq": ["triton_v2"]`

⚠️  Note that getting error in loading checkpoint in accelerate_launch.py that will either be addressed with updated accelerate version and removal of lm_head in accelerate_launch.py or need to fix to load model appropriately on the set device.

### Was the PR tested
